huaweicloud/services/acceptance/vpc/data_source_huaweicloud_vpcs_test.go

```diff
@@ -166,22 +164,21 @@ func TestAccVpcsDataSource_byAll(t *testing.T) {
 	})
 }
 
-func testAccDataSourceVpcs_byAll(rName, cidr, enterpriseProjectID string) string {
+func testAccDataSourceVpcs_byAll(rName, cidr string) string {
 	return fmt.Sprintf(`
 %s
 
 data "huaweicloud_vpcs" "test" {
   id                    = huaweicloud_vpc.test.id
   name                  = huaweicloud_vpc.test.name
   cidr                  = huaweicloud_vpc.test.cidr
-  enterprise_project_id = "%s"
   status                = "OK"
 
   depends_on = [
     huaweicloud_vpc.test
   ]
```

Comment:
```suggestion
```
`depends_on` is unnecessary.

Reply:
The test may be failed if no huaweicloud_vpc resource is previously created.